grpc-locations/src/main/resources/application.yml

```diff
@@ -0,0 +1,123 @@
+quarkus:
+  application:
+    name: grpc-locations
+  log:
+    level: INFO
+    category:
+      "io.quarkus.sample.superheroes":
+        level: DEBUG
+    console:
+      format: "%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId}, sampled=%X{sampled} [%c{2.}] (%t) %s%e%n"
+      level: INFO
+      darken: 1
+  http:
+    port: 8089
+    test-port: 0
+  grpc:
+    server:
+      use-separate-server: false
+      enable-reflection-service: true
```

Comment:
That's the default in dev. What's your reason for enabling the reflection service in test and prod?

Reply:
I thought because this was a sample app it might be a good thing. Similar to how in the other rest services we enable swagger-ui in prod.

Reply:
ok